lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -137,10 +137,12 @@ export class ServerRequestParameters {
     str.push(`x-client-SKU=${this.xClientSku}`);
     str.push(`x-client-Ver=${this.xClientVer}`);
 
-    str.push("prompt=" + encodeURI(this.promptValue));
+    if (this.promptValue) {
```

Comment:
is there a valid set of valid promptValues ?

Reply:
Yes, this came as a part of a different PR #618  which you just reviewed.